crypto-jni/src/lib.rs

```diff
@@ -408,3 +465,216 @@ pub extern "system" fn Java_com_concordium_sdk_crypto_CryptoJniNative_generateCo
 
     return CryptoJniResult::Ok(payload).to_jstring(&env);
 }
+
+type SerializeParamResult = CryptoJniResult<String>;
+
+#[no_mangle]
+#[allow(non_snake_case)]
+/// The JNI wrapper for serializing ia parameter for a recieve function according to a specified Schema.
+/**
+ * `parameter` The parameter to serialize. `parameter` must be a properly initalized `java.lang.String` that is non-null. The parameter must be valid JSON according to the provided schema.
+ * `contractName` The name of the contract. `contractName` must be a properly initalized `java.lang.String` that is non-null.
+ * `methodName` The name of the method. `methodName` must be a properly initalized `java.lang.String` that is non-null.
+ * `schemaBytes` The bytes of the schema. `schemaBytes` must be a properly initalized `byte[]` containing bytes representing a valid schema. The schema must match the provided `contractName`.
+ * `schemaVersion` The version of the Schema. `schemaVersion` must be an integer representing a valid `com.concordium.sdk.transactions.smartcontracts.SchemaVersion`.
+ * `verboseErrors` Whether errors are returned in verbose format or not, can be useful when debugging why serialization fails.
+ */
```

Comment:
This is unusual formatting for Rust. You should just use markdown in doc comments